app/models/presentation.rb

```diff
@@ -3,12 +3,14 @@ class Presentation < ApplicationRecord
   belongs_to :event
 
   validates :topic,     presence: true
-  validates :duration,  presence: true, numericality: true
   validates :person,    presence: true, unless: :presenter
   validates :presenter, presence: true, unless: :person
   validates :event,     presence: true
+  validates :duration,  presence: true, numericality: {
```

Comment:
Current error messages being displayed in the form flash include:
Duration must be less than or equal to 15
Duration must be greater than 0

I feel pretty good about these. The application is particularly user friendly at this point and my thoughts kind of lead me to think if we are going to set a more friendly message here we should do it for all the validation errors. 

Ultimately I think it's a good idea but in the state the application is in right now I don't think it's the right time.